reference/system-databases/cluster-hardware.md

```diff
@@ -45,18 +45,18 @@ desc cluster_hardware;
 {{< copyable "sql" >}}
 
 ```sql
-select * from cluster_hardware where device_type='cpu' and device_name='cpu' and name like '%cores';
+select * from information_schema.cluster_hardware where device_type='cpu' and device_name='cpu' and name like '%cores';
 ```
 
 ```
 +------+-----------------+-------------+-------------+--------------------+-------+
 | TYPE | INSTANCE        | DEVICE_TYPE | DEVICE_NAME | NAME               | VALUE |
 +------+-----------------+-------------+-------------+--------------------+-------+
-| tidb | 127.0.0.1:10080 | cpu         | cpu         | cpu-logical-cores  | 8     |
-| tidb | 127.0.0.1:10080 | cpu         | cpu         | cpu-physical-cores | 4     |
-| pd   | 127.0.0.1:2379  | cpu         | cpu         | cpu-logical-cores  | 8     |
+| tidb | 0.0.0.0:4000    | cpu         | cpu         | cpu-logical-cores  | 4     |
```

Comment:
Both cpu-logical-cores and cpu-physical-cores is 4?

Reply:
I run it in the Virtual Machine of the Virtual Box，so it show both of them are same.